Central-Server/API/internal/database.py

```diff
@@ -0,0 +1,54 @@
+"""This file exposes functions that abstract certain database functions"""
+import os
+import collections
+
+import psycopg2

```

Comment:
STYLE: We should follow [python import guidelines](https://peps.python.org/pep-0008/#imports)

Reply:
nvm